src/main/java/ml/shifu/shifu/util/CommonUtils.java

```diff
@@ -3112,4 +3154,43 @@ public static boolean isHadoopConfigurationInjected(String key) {
                 || key.startsWith("io") || key.startsWith("hadoop") || key.startsWith("yarn") || key.startsWith("pig")
                 || key.startsWith("hive") || key.startsWith("job");
     }
+    
+    /**
+     * Merge two one-dimension matrix into one with add operation
+     * @param a
+     * @param b
+     * @return
+     */
+    public static double[] merge(double[] a, double[] b) {
+        if (a == null || b == null || a.length != b.length) {
+            throw new RuntimeException("Input is invalid. a is " + Arrays.toString(a) + 
```

Comment:
IllegalArgumentException is better than RuntimeException.

Best practice should not use RuntimeException instread of a cutomized RuntimeException